roles/config/tasks/6_domains.yml

```diff
@@ -5,8 +5,8 @@
     server_url: "{{ satellite_server_url }}"
     name: "{{ item.name }}"
     description: "{{ item.description | default(omit) }}"
-    locations: "{{ item.locations }}"
-    organizations: "{{ item.organizations }}"
-    dns_proxy: "{{ satellite_fqdn | default(omit) }}"
+    locations: "{{ item.locations | default(omit) }}"
+    organizations: "{{ item.organizations | default(omit) }}"
+    dns_proxy: "{{ item.dns_proxy | default(omit) }}"
```

Comment:
I see that `dns_proxy` can be defined in `satellite_subnets`, but I don't see `dns_proxy` as a key directly defined within `satellite_domains`?

Have a functioning example of this working?

Reply:
TBH I don't know what the `dns_proxy` is for, but it doesn't work for our deploy for multiple reasons:

- it fails without gathering facts, as `satellite_fqdn` defaults to `ansible_fqdn`, applies to some other defaults too, which I commented out
- it fails without the DNS proxy things be configured, whatever that is, but we don't have/need that
- it expects all domains to have same `dns_proxy`, which doesn't seem legit to me